src/lineup/internal/LineUpColors.ts

```diff
@@ -7,19 +7,7 @@ export default class LineUpColors {
    * Map that assigns each selection ID a color, which is used as color for columns
    */
   private readonly colorMap = new Map<number, {color: string, items: number}>();
-  private colors: string[];
-
-  init(ranking: Ranking) {
-    const colors = scale.category10().range().concat(scale.category20().range().filter((_d,i) => i % 2 === 1));
-    // remove colors that are already in use from the list
-    ranking.flatColumns.forEach((d) => {
-      const i = colors.indexOf(d.color);
-      if (i > -1) {
-        colors.splice(i, 1);
```

Comment:
don't we need to remove the colors which are already in use anymore?

Reply:
In LineUp v3.2 the `color` property was removed from the `Column`. Only `NumericalColumn` support colors in form of `colorMapping`. The difference here: the color is determined on a per row basis, as each bar can have a different color. When initializing the columns, we don't have rows and _one_ color. Hence, we can remove the check for duplicate colors.

Reply:
Side note: For numerical columns every of the 20 colors are still used once, but it will not be checked what colors already added columns have. 

![grafik](https://user-images.githubusercontent.com/5851088/65020725-065d1f80-d92e-11e9-863f-6c2e82a3f502.png)